bigtable-hbase-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/filters/ColumnPaginationFilterAdapter.java

```diff
@@ -44,22 +45,18 @@ public RowFilter adapt(FilterAdapterContext context, ColumnPaginationFilter filt
       throws IOException {
     if (filter.getColumnOffset() != null) {
       byte[] family = context.getScan().getFamilies()[0];
+      ByteString startQualifier = ByteString.copyFrom(filter.getColumnOffset());
       // Include all cells starting at the qualifier scan.getColumnOffset()
       // up to limit cells.
       return createChain(
           filter,
-          RowFilter.newBuilder()
-              .setColumnRangeFilter(
-                  ColumnRange.newBuilder()
-                      .setFamilyName(Bytes.toString(family))
-                      .setStartQualifierClosed(
-                          ByteString.copyFrom(filter.getColumnOffset()))));
+          F.qualifier().range(Bytes.toString(family))
```

Comment:
I know I LGTM'ed this, but it looking at its usage makes me second guess that decision. How would you feel about amending it to:

`F.qualifier().exactFamilyMatch("family").startClosed(...).endOpen(...)`

Or something else, `range(family)` is too misleading

Reply:
SGTM